content/2020/August/README.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "August PythonPune meetup 2020"
+date: 2020-11-21T17:26:16+05:30
+event_date: 2020-08-30T15:30:00+05:30
+url: event/august-pythonpune-meetup-2020
+author: Bhavin Gandhi
+featured_image:
+categories:
+- Event
+tags:
+- Benchmarking
+- Python Internals
+- cffi
+---
+
+  * [Event page on meetup.com](https://www.meetup.com/PythonPune/events/272520240/)
+  * [Community Newsletter](./community_news.md)
+  * [Recording](https://youtu.be/nw-lvSpnii4)
+
+## Distributed system analysis made easy with PBench
+  * **Speaker**: Riya
+    ([LinkedIn](https://www.linkedin.com/in/riya-17/))
+  * **Description**: Riya introduces everyone to the PBench tool. She
+    talks about the need of collecting benchmarking results in a
+    standard way. She gives a demo of the PBench tool along with its
+    architecture.
+  * **References**:
+    * [Slides](https://slides.com/riya_17/neural-artistic-style-transfer-6efa60)
+    * [PBench](https://distributed-system-analysis.github.io/pbench/)
+    * [Video recording](https://youtu.be/x1ORnZp_oRI)
+    * [Proposal](https://github.com/pythonpune/meetup-talks/issues/107)
+
+> ***NOTE**: You can reach out to the speaker on the proposal page if
+> you want to discuss anything about this talk.*
```

Comment:
@bhavin192  I think it would be enough to show this note only one time (probably at the end) in a page. What do you think?

Reply:
I had a thought about that, but one might come to a page from somewhere like YouTube and they might not go to the end of the page. Some of the talks might not even have proposals. We can still experiment with the position and style of that note for sure. 
How about have it somewhere after the description of the talks as normal line only (non quoted)?

Reply:
It's true that not all reader reach to the end to read this note. It just that we don't have very long post, one could see same NOTE as it feels like it is repeated twice.
As this note is important, we could place it per talk considering not all talks have proposals.

Current position and format is fine to me.